src/scss/colors.scss

```diff
@@ -0,0 +1 @@
+$white: #ffffff;
```

Comment:
There already is one (surprisingly empty) `colors.scss` file, but I think it should be moved under `./src/scss/admin/colors.scss`

Reply:
I didn't move the file on purpose because I want this one to be the shared one, between attendee and admin. There is no point in adding `$white: #FFFFFF` in both admin and attendee `colors.scss` files